layout.pretty/Relay_SPST_Omron_G6DN.kicad_mod

```diff
@@ -0,0 +1,30 @@
+(module Relay_SPST_Omron_G6DN (layer F.Cu) (tedit 5F2890A6)
+  (descr "SPST Slim Power Relay for 5 A Switching")
+  (tags "Relay SPST Omron 4Pin")
+  (fp_text reference K1 (at 20.4397 1.16824 -90) (layer F.SilkS)
```

Comment:
The Refdes here should not be K1 or anything specific. It should be `**REF` (change this in footprint properties)